### PR TITLE
CLI: check for conflicting commands.

### LINKED ISSUE #1982: CLI commands should be mutually exclusive, only single command should be allowed per each run
## Description

It is possible to pass multiple command/action switches when running `rnp` or `rnpkey` CLI program,
but only the last one is actually executed.

## Steps to Reproduce
```
rnpkeys --generate-key --import --revoke-key --list-keys
```
or
```
rnp --sign --verify --decrypt --list-packets ~/.rnp/pubring.gpg
```

## Expected Behavior

CLI tools should print an appropriate error message.

## Actual Behavior

The last passed command is executed, in this case `--list-keys` and `--list-packets`.
This behavior could lead to accidental unexpected errors when using CLI programs.

### PR DESCRIPTION
**Pull request by antonsviridenko**
This PR adds new error message to `rnp` and `rnpkeys` when multiple conflicting commands are passed on a command line.

Closes #1982

### CODE REVIEW

File: src/rnp/rnp.cpp [hunk 1]
```diff
@@ -353,6 +353,11 @@ setcmd(rnp_cfg &cfg, int cmd, const char *arg)
         break;
     }
 
+    if (cfg.has(CFG_COMMAND) && cfg.get_int(CFG_COMMAND) != newcmd) {
```
[the remark this answers is not in the document]
Comment (ni4) [reply src/rnp/rnp.cpp:356]: @antonsviridenko Ah, missed that. In this way should work fine.

**Review by ni4 — APPROVED** [review]
LGTM, thanks!

**Review by ronaldtse — APPROVED** [review]

**Comment by antonsviridenko** [comment]
@ni4  @ronaldtse what task can I work on next?

**Comment by ni4** [comment]
@antonsviridenko #1057 #1277 #1777 #1865 #1997 #2001 or any other issue you'll find interesting, prioritizing v0.18.0 milestone.

**Pull request merged**